### DISCUSSION TITLE
遇到一个bug 上传大内存包时不成功也不进行报错 (A bug has occurred where uploading a large memory package fails and no error is reported)

### DISCUSSION THREAD

**Discussion by a67757954** [verdaccio/verdaccio#4850] [Q&A]
1、6.8mb的文件提示上传成功，但是在verdaccio 里面没有进行显示，按照道理来说应该有错误进行报错的呢

> 1. The 6.8mb file is uploaded successfully, but it is not displayed in Verdaccio. Logically, there should be an error.

![image](https://github.com/user-attachments/assets/ce325a15-dd50-40c3-b7ae-6f2dbfc91727)

**Comment by juanpicado** [comment]
@a67757954 advice please use a translator, it does not matter if English looks clunky or inaccurate but in other languages than English hardly will get help.

Anyhow, the info provided is not good enough for understand the issue.

**Reply by marcfbe** [reply]
He's publishing the same version twice. This should result in an error but does not 🤷
